controllers/app.controller.js

```diff
@@ -12,8 +13,11 @@ exports.getTopics = (req, res) => {
     })
 };
 
-exports.getArticlesById = (req, res) => {
-    findArticlesById(req.params.article_id).then((data) => {
+exports.getArticlesById = (req, res, next) => {
+    findArticlesById(req.params.article_id)
```

Comment:
I think this would be more readable if you destructured the article_id from the req.params before passing it into the function 🧐